lib/fictium/exporters/api_blueprint_exporter/resource_formatter.rb

```diff
@@ -0,0 +1,31 @@
+module Fictium
+  class ApiBlueprintExporter
+    class ResourceFormatter
+      def format(resource)
+        [build_header(resource), build_actions(resource)]
+          .map(&:strip)
+          .select(&:present?)
+          .join("\n\n")
+      end
+
+      private
+
+      def build_header(resource)
+        header = "# #{resource.name&.pluralize&.humanize} [#{resource.base_path}]"
+        header += "\n#{resource.summary.capitalize}\n" if resource.summary.present?
+        header += "\n#{resource.description}\n" if resource.description.present?
+        header
+      end
+
+      def build_actions(resource)
+        resource.actions.map do |action|
+          action_formatter.format(action)
+        end.select(&:present?).join("\n\n")
```

Comment:
:eyes: You often are using `select(&:present?).join("\n\n")`, you can delegate it in BaseExporter